examples/connectivity/plot_adhd_covariance.py

```diff
@@ -8,11 +8,26 @@
  
 plotted_subject = 0  # subject to plot
 
+import numpy as np
 
 import matplotlib.pyplot as plt
+
+from nilearn import plotting, image
 from nilearn.plotting import cm
 
 
+def plot_connectome(cov, atlas_maps, **kwargs):
+    """Plot connectome given a covariance matrix and atlas maps"""
+    imgs = image.iter_img(atlas_maps)
+    regions_coords = np.array([
```

Comment:
Could we do a for loop rather than a map: this is going to confuse beginners.

Also, a comment saying why this code is useful would help.

Finally, I think that I would prefer not having a function defined for this, but simply code in the example, also in the interest of simplicity.